run_tests.sh

```diff
@@ -40,7 +59,7 @@ testrepo () {
   git clone https://github.com/dcrlabs/bug-free-happiness $TMPDIR/test-data-repo
   tar xvf $TMPDIR/test-data-repo/stakedb/test_ticket_pool.bdgr.tar.xz -C ./stakedb
 
-  env GORACE='halt_on_error=1' go test -v -race ./...
+  env GORACE='halt_on_error=1' go test -v -race ./... $ALLMODULES
```

Comment:
So I understand, the `./...` tests the packages belonging to the root module, and then we have to list the other modules?

Reply:
nvm my comment, doing that would not test packages like `explorer` that are not modules.